docs/src/componentDocs/Header/markdown/HeaderExamples.mdx

```diff
@@ -1,180 +1,35 @@
-import Box from '@mui/material/Box';
-import headerImageExample from '../images/headerImage.png';
-import headerBackgroundExample from '../images/headerBackgroundImage.png';
-import headerActionsExample from '../images/headerActionsImage.png';
-import headerSearchExample from '../images/headerSearchImage.png';
-import headerCollapsedExample from '../images/headerCollapsedImage.png';
-import headerExpandedExample from '../images/headerExpandedImage.png';
+import {Header} from '../examples/Header';
+import {HeaderWithBackground} from '../examples/HeaderWithBackground';
+import {HeaderWithActions} from '../examples/HeaderWithActions';
+import {HeaderWithSearch} from '../examples/HeaderWithSearch';
+import {HeaderWithExpandCollapse} from '../examples/HeaderWithExpandCollapse';
 
 ## Header
 
-A basic `Header` component is used at the top of the screen and can show a title and (optional) subtitle. A third line of text can also be configured using the `info` prop, but it will only be visible when the `Header` is expanded (see below). 
+A basic `Header` component is used at the top of the screen and can show a title and (optional) subtitle. A third line of text can also be configured using the `info` prop, but it will only be visible when the `Header` is expanded (see below).
```

Comment:
Following the conventions, the component name should be written inside angle braces `<Header>`

Reply:
I missed this in earlier PR.